### PR TITLE
Followed artists tab on user page: filter by tag

### PR DESCRIPTION
**Pull request by ycanardeau**
#511

### CODE REVIEW

File: VocaDbWeb/Scripts/Repositories/UserRepository.ts [hunk 1]
```diff
@@ -123,14 +123,19 @@ module vdb.repositories {
 
 		public getFollowedArtistsList = (
 			userId: number,
-			paging: dc.PagingProperties, lang: string,
+            paging: dc.PagingProperties, lang: string,
```
Comment (riipah) [inline VocaDbWeb/Scripts/Repositories/UserRepository.ts]: Hmm, spacing seems to have changed here. It should be specified in .editorconfig already.

File: VocaDbWeb/Scripts/ViewModels/User/FollowedArtistsViewModel.ts [hunk 2]
```diff
@@ -35,7 +39,8 @@ module vdb.viewModels.user {
 		public page = ko.observableArray<dc.RatedSongForUserForApiContract>([]); // Current page of items
 		public paging = new ServerSidePagingViewModel(20); // Paging view model
 		public pauseNotifications = false;
-		public resources = ko.observable<any>();
+        public resources = ko.observable<any>();
```
Comment (riipah) [inline VocaDbWeb/Scripts/ViewModels/User/FollowedArtistsViewModel.ts]: Could you correct all these indentation changes? Thanks.

**Comment by riipah** [comment]
Seems okay. Thank you very much. It's been too long after working on the frontend, I can't be sure if this is all that's needed, but assuming you tested it, then it should be fine. There was some unnecessary indentation changes that you could fix. I'm impressed though how someone has figured out the codebase so quickly.

File: VocaDbWeb/Scripts/ViewModels/User/FollowedArtistsViewModel.ts (hunk 2 above)
Comment (ycanardeau) [reply VocaDbWeb/Scripts/ViewModels/User/FollowedArtistsViewModel.ts]: fixed

**Comment by riipah** [comment]
Thanks, this should be fine now.

**Pull request merged**